### PR TITLE
Adding Issue & pull request templates

### PR DESCRIPTION
**Pull request by rajitha1998**
Fixes - #6 #18  
* pull request template
* Issue template
* Minor unwanted code removal

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Fixes #
+
+## Proposed Changes
+
+  -
```
Comment (janethavi) [inline .github/PULL_REQUEST_TEMPLATE.md:5]: @rajitha1998 Shall we add a `Screenshots` section to the PULL_REQUEST_TEMPLATE?

**Review by janethavi — CHANGES_REQUESTED** [review]

**Review by hasalfernando — APPROVED** [review]

**Comment by janethavi** [comment]
@rajitha1998 Can u please mention the issue number:- 18 in the description because you have fixed it too?

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (rajitha1998) [reply .github/PULL_REQUEST_TEMPLATE.md:5]: sure

Comment (rajitha1998) [reply .github/PULL_REQUEST_TEMPLATE.md:5]: @janethavi  added screenshots section

**Comment by janethavi** [comment]
Nice work @rajitha1998 :100:

**Pull request merged**